internal/runtime/init/init.go

```diff
@@ -109,6 +109,22 @@ type Descriptor struct {
 	Path string
 }
 
+// removeDuplicatePaths removes duplicate file paths in order
+func removeDuplicatePaths(paths []string) []string {
+	pathMap := make(map[string]bool)
+	result := []string{}
+
+	for _, path := range paths {
+		if pathMap[path] {
+			continue
+		}
+		pathMap[path] = true
+		result = append(result, path)
+	}
+
+	return result
```

Comment:
nit: sort the result so that the order of the paths is deterministic; otherwise file load errors will differ across runs which is not good for the UX.

Reply:
Good idea! If I sort the file though, that means the order of the files are not retained which might affect declaration of terms since it follows one after the other. Will this be a problem?